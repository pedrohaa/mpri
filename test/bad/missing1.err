File "bad/missing1.f", line 12, characters 2-65:
A case for the data constructor Nil is missing.
Hypotheses:
var = a
