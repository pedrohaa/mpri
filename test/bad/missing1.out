forall a. list a -> a
